source/public_collections_advanced_search.md

```diff
@@ -0,0 +1,103 @@
+# Public Collections Advanced Search
+
+**_Please note that the search capability is a work in progress, and we plan to
+add a more user-friendly syntax in future releases._**
+
+### The Default Search Behaviour
+
+When you enter a term into the search bar without specifying a field, your
+query will return results where that term, regardless of capitalization, can be
+found in:
+- The name of the survey, collection, question, block, or template;
+- The owner's username;
+- The description;
+- The summary, which contains all question labels and languages;
+- The name of any assigned tag;
+- The object's UID.
+
+For example, a default search with the term: "_examples_", will result in the
+following:
+
+```
+name__icontains:examples OR owner__username__icontains:examples OR
+settings__description__icontains:examples OR summary__icontains:examples OR
+tags__name__icontains:examples OR uid__icontains:examples
+```
+
+![examples](/images/public_collections_advanced_search/advanced_search_1.png)
+
+### Valid Search Field Operators
+
+The field operator is the value after the last double-underscore in the search
+field, i.e. `__icontains`.
```

Comment:
I'm feeling that this should be "e.g." ("exempli gratia" / "for example"), but I guess if it's referring to the query underneath the previous heading, maybe "i.e." ("id est" / "that is") is correct.
![image](https://user-images.githubusercontent.com/2085013/110400917-8c9b8300-8046-11eb-9e1d-0733d1700c26.png)

Reply:
@jnm I always get confused which of those is best to use when 😂 should I just leave it as is then?